packages/agreement-process/src/services/agreementService.ts

```diff
@@ -1176,3 +1190,33 @@ function maybeCreateSetToMissingCertifiedAttributesByPlatformEvent(
   }
   return undefined;
 }
+
+// eslint-disable-next-line max-params
+async function addContractOnFirstActivation(
```

Comment:
Moved this here from `agreementSubmissionProcessor.ts` because now I reuse it also for activation